sdk/internal/temporal/resource.go

```diff
@@ -62,12 +89,12 @@ func (er *Resource[TResource, TState]) Get(state TState) (TResource, error) {
 			if !er.acquiring {
 				// If another thread/goroutine is not acquiring/updating the resource, this thread/goroutine will do it
 				er.acquiring, acquire = true, true
+				er.lastAttempt = now
```

Comment:
I think this should have also been removed in the reversion?

Reply:
You're quite right! My bad.